lms/static/sass/base/_headings.scss

```diff
@@ -121,5 +121,9 @@ $headings-base-color:                       $gray-d2;
         font-weight: $headings-font-weight-normal;
         // override external modules and xblocks that use inline CSS
         text-transform: initial;
+        &.h3--discussion-module {
```

Comment:
This name doesn't follow our usual conventions. I'd call this ``discussion-module-title``.